tests/compile-fail/arithmetic.rs

```diff
@@ -0,0 +1,31 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+#![deny(integer_arithmetic, float_arithmetic)]
+#![allow(unused, shadow_reuse, shadow_unrelated, no_effect)]
+fn main() {
+    let i = 1i32;
+    1 + i; //~ERROR integer arithmetic detected
+    i * 2; //~ERROR integer arithmetic detected
+    1 % //~ERROR integer arithmetic detected
```

Comment:
No, the expression is continued on the next line on purpose. This tests that only the outermost expression is linted.